compiler/infra/Assert.cpp

```diff
@@ -40,39 +40,37 @@
 
 void OMR_NORETURN TR::trap()
    {
-   static char * noDebug = feGetEnv("TR_NoDebuggerBreakPoint");
-   if (noDebug)
+   char * noDebug = feGetEnv("TR_NoDebuggerBreakPoint");
+   if (!noDebug)
       {
-      exit(1337);
-      }
-   else
-      {
-      static char *crashLogOnAssume = feGetEnv("TR_crashLogOnAssume");
+      char *crashLogOnAssume = feGetEnv("TR_crashLogOnAssume");
       if (crashLogOnAssume)
          {
          //FIXME: this doesn't work on z/OS
          *(volatile int*)(0) = 0; // let crashlog do its thing
```

Comment:
This comment is stale. You can read from location 0 on z/OS as the page is not protected, but you can't write. We should remove the comment.

Reply:
Will remove! (probably tomorrow)